test/primus.test.js

```diff
@@ -766,3 +773,9 @@ describe('Primus', function () {
     });
   });
 });
+
+function stub(obj, prop) {
+  var stubbed = obj[prop];
```

Comment:
Was just copying the style of the rest of the tests; we're going ES6 for all new code?